system/modules/isotope/languages/en/tl_iso_payment.xlf

```diff
@@ -303,10 +303,10 @@
         <source>Your project password for sofortüberweisung.de</source>
       </trans-unit>
       <trans-unit id="tl_iso_payment.saferpay_accountid.0">
-        <source>Saferpay Account-ID</source>
+        <source>Saferpay Account ID (CustomerId-TerminalId)</source>
       </trans-unit>
       <trans-unit id="tl_iso_payment.saferpay_accountid.1">
-        <source>Please enter your unique Saferpay account id.</source>
+        <source>Please enter your unique Saferpay account id. It should be in the form CustomerId-CustomerId</source>
```

Comment:
Shouldn't it be `CustomerId-TerminalId` here as well?

Reply:
Oh yes, you are right it should be `CustomerId-TerminalId`. My mistake

Reply:
Please update the Pull Request then 😄

Reply:
Oh, I've updated it.